colcon-test/README.md

```diff
@@ -0,0 +1,84 @@
+# colcon-test
+
+This action runs `colcon test` with a label.
```

Comment:
Considering this conversation, how about this?
https://github.com/autowarefoundation/autoware-github-actions/pull/49#discussion_r800471009

```suggestion
This action runs `colcon test` with labels specified in a regex format.
```

Reply:
Good. Thanks!